frontend/hub/administration/repositories/RepositoryPage/RepositoryDetails.tsx

```diff
@@ -22,28 +22,30 @@ export function RepositoryDetails() {
   const getPageUrl = useGetPageUrl();
   const { repository } = useOutletContext<{ repository: Repository }>();
 
-  const {
-    data: distroData,
-    error: distroError,
-    refresh,
-  } = useGet<{ results: { name: string; client_url: string }[] }>(
-    params.id ? pulpAPI`/distributions/ansible/ansible/?name=${params.id}` : ''
-  );
+  const { data: distroData, error: distroError } = useGet<{
+    results: { name: string; client_url: string }[];
+  }>(params.id ? pulpAPI`/distributions/ansible/ansible/?name=${params.id}` : '');
+
   const { data: remoteData, error: remoteError } = useGet<Task>(
-    params.id ? pulpAPI`/remotes/ansible/collection/${parsePulpIDFromURL(repository.remote)}/` : ''
+    params.id ? pulpAPI`/remotes/ansible/collection${parsePulpIDFromURL(repository.remote)}/` : ''
```

Comment:
There has to be a `/` to get the correct result for others. If there's no repository.remote it should return an error instead of failing.